File: app/src/main/java/com/roman/batsu/ui/model/Lessons.java
package com.roman.batsu.ui.model;

public class Lessons {

    private String lesson_name;
    private String count;
    private String lesson_type;
    private String teacher_surname;

    public String getLesson_name() {
        return lesson_name;
    }


    public String getCount() {
        return count;
    }


    public String getLesson_type() {
        return lesson_type;
    }

    public String getTeacher_surname() {
        return teacher_surname;
    }

    @Override
    public String toString() {
        return lesson_name + ", count = " + count;
    }

}

File: app/src/main/java/com/roman/batsu/ui/model/Lessons.kt
package com.roman.batsu.ui.model

import com.google.gson.annotations.SerializedName

class Lessons {
    @SerializedName("lesson_name")
    val lessonName: String? = null
    @SerializedName("count")
    val count: String? = null

    @SerializedName("lesson_type")
    val lessonType: String? = null

    @SerializedName("teacher_surname")
    val teacherSurname: String? = null

    override fun toString(): String {
        return "$lessonName, count = $count"
    }
}